app/views/users/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<h1>New User Registration</h1>
+<p>Enter details below</p>
```

Comment:
This line is self-explanatory and I think we should consider removing it during a refactor. Thoughts?

Reply:
I agree, not necessary

Reply:
also agree. "below" consists of the entire page, which is just one form.

Reply:
I removed line 2